source/usage-examples/updateMany.txt

```diff
@@ -3,3 +3,56 @@ Update Multiple Documents
 =========================
 
 .. default-domain:: mongodb
+
+You can update multiple documents in a collection by using the ``UpdateMany()``
+method.
+
+The following example passes a query filter and an update parameter to the
+``UpdateMany()`` method, which matches documents in the
+``listingsAndReviews`` collection where the value of the ``address.market``
+field is "Sydney" and multiplies the ``price`` field in the matching
+documents by a factor of *1.15*:
+
+.. include:: /includes/run-example-tip.rst
+
+.. literalinclude:: /code-snippets/updateMany.go
+   :start-after: begin updatemany
+   :end-before: end updatemany
+   :emphasize-lines: 5
+   :language: go
+   :dedent:
+
+Click here **<TODO: github link to file>** to see a fully runnable example.
```

Comment:
I assume similar to https://github.com/mongodb/docs-golang/pull/5#discussion_r676970318 this link, and the TODO link below will be updated before publishing.